bee-hive/bee_hive/__init__.py

```diff
@@ -1,6 +1,3 @@
-
-__all__ = [
-    "Agent",
-    "Workflow",
-    "Interface",
-]
+# SPDX-License-Identifier: Apache-2.0
+from .workflow import Workflow
+from .bee_agent import BeeAgent
```

Comment:
save_agent, restore_agent and remove_agent in agent.py are necessary.

Reply:
You mean you want those added to export?
Is there a test case you can describe to verify?

Reply:
No, not at this moment.  These are used to create the agent first and use it in the workflow later case.

Reply:
So are we ok to merge and then you can add when you are ready?

Reply:
I went ahead and added them via a suggest&commit (of course, now there's a merge conflict that is blocking the merge :smiling_face_with_tear: )